src/ospf_fmt_plug.c

```diff
@@ -39,7 +39,7 @@ john_register_one(&fmt_ospf);
 #define TAG_LENGTH              (sizeof(FORMAT_TAG) - 1)
 #define ALGORITHM_NAME          "HMAC-SHA-X 32/" ARCH_BITS_STR
 #define BENCHMARK_COMMENT       ""
-#define BENCHMARK_LENGTH        0x107
+#define BENCHMARK_LENGTH        7
```

Comment:
This looked OK at first, but no. The first two test vectors are different speed. Please use `0x507` here and add a comment. Perhaps also add a FIXME comment requesting reporting of tunable costs.
```
[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=ospf   
Will run 256 OpenMP threads
Benchmarking: ospf, OSPF / IS-IS [HMAC-SHA-X 32/64]... (256xOMP) DONE
Warning: "Many salts" test limited: 7/256
Many salts:     12336K c/s real, 53119 c/s virtual
Only one salt:  8962K c/s real, 81945 c/s virtual

[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ../run/john -test -form=ospf
Will run 256 OpenMP threads
Benchmarking: ospf, OSPF / IS-IS [HMAC-SHA-X 32/64]... (256xOMP) DONE
Warning: "Many salts" test limited: 7/256
Many salts:     12440K c/s real, 53136 c/s virtual
Only one salt:  9039K c/s real, 82003 c/s virtual

[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ./john -mask -max-run-time=10 pw-ospf1
Using default input encoding: UTF-8
Loaded 1 password hash (ospf, OSPF / IS-IS [HMAC-SHA-X 32/64])
Will run 256 OpenMP threads
Using default mask: ?1?2?2?2?2?2?2?3?3?3?3?d?d?d?d
Press 'q' or Ctrl-C to abort, almost any other key for status
0g 0:00:00:10  0g/s 6738Kp/s 6738Kc/s 6738KC/s ncc4zaaaaaa1111..r8hzxaaaaaa1111
Session stopped (max run-time reached)
[solar@localhost run]$ GOMP_CPU_AFFINITY=0-255 GOMP_SPINCOUNT=10000 ./john -mask -max-run-time=10 pw-ospf2
Using default input encoding: UTF-8
Loaded 1 password hash (ospf, OSPF / IS-IS [HMAC-SHA-X 32/64])
Will run 256 OpenMP threads
Using default mask: ?1?2?2?2?2?2?2?3?3?3?3?d?d?d?d
Press 'q' or Ctrl-C to abort, almost any other key for status
0g 0:00:00:10  0g/s 5075Kp/s 5075Kc/s 5075KC/s Hfgnpaaaaaa1111..Ulf9paaaaaa1111
Session stopped (max run-time reached)
[solar@localhost run]$ cat pw-ospf1
$ospf$1$02010030ac10001400000000000000020000011454ee4518ffffff00000a120100000028c0a86f14c0a86f0aac10000a$e59ba2c56a2c0429ebe72a194e4b54c250cac1a3
[solar@localhost run]$ cat pw-ospf2
$ospf$2$0201002cac10000a00000000000000020000012054f4c8adffffff00000a120100000028c0a86f0a00000000$508a1abffb5b4554e1aa46eb053bca7105c3e8f6fece4c945f0a0020edb054ec
```